src/main/java/com/amihaiemil/docker/SslHttpClient.java

```diff
@@ -59,16 +62,37 @@
 
     /**
      * Ctor.
-     * @param certs Path to the folder containing the following certificates:
-     *  ca.pem, cert.pem and key.pem.
+     * @param keys Path to the keystore.
+     * @param trust Path to the truststore.
+     * @param storePwd Password for the keystore.
+     * @param keyPwd Passphrase for the key.
      */
-    SslHttpClient(final Path certs) {
-        this(
-            HttpClients.custom()
-                .setMaxConnPerRoute(10)
-                .setMaxConnTotal(10)
-                .build()
-        );
+    SslHttpClient(
+        final Path keys, final Path trust,
+        final char[] storePwd, final char[] keyPwd) {
+        this(() -> {
+            try {
+                return HttpClients.custom()
+                    .setMaxConnPerRoute(10)
+                    .setMaxConnTotal(10)
+                    .setSSLContext(
```

Comment:
@llorllale Are you sure this ``custom()`` HttpClient is enough to use double-way HTTPS? I'm not so experienced with Apache HttpClient and the Spotify fellas did it quite differently :D

Reply:
@amihaiemil I haven't tested it and never done it before. I went by these links:

* https://stackoverflow.com/a/32513368/1623885
* https://stackoverflow.com/a/46821977/1623885